eng/scripts/Cadl-Project-Sync.ps1

```diff
@@ -0,0 +1,152 @@
+[CmdletBinding()]
```

Comment:
No we would need to add in some "offline" mode.  Right now it will always try to pull the changes in for the commit sha since it doesn't know if the local files are up to date or not.

we could do this by some flag in the cadl-location.yaml such as `commit: local` to indicate don't sync just use my local files.

Do you mind filing an issue for this and we can address after this PR unless you think its a requirement for v1.  Trying to keep this to MVP as the dpg timelines are very sensitive right now.

Reply:
We could potentially add some configuration option that allows folks to point at a local specs repo clone as I do think that will be an interesting scenario even outside of the offline scenario.